### ISSUE TITLE
Need language filtering

### ISSUE THREAD

**Issue by dragosv** [phrase/phraseapp-client#133]
Need to be able to retrieve only the languages specified by the CLI, instead of all, so a parameter like --lang=fr,es or similar should be used. This will also improve PhraseApp compatibility with Serge.io (https://serge.io/)

**Comment by dragosv** [comment]
This will also improve compatibility with other translation software CLI (Mojito, Zanata and a few commercial ones)

**Comment by theSoenke** [comment]
Do you need this option when running `phraseapp pull`? Could you maybe describe your use case a little bit more in detail? You could also only specify e.g. fr and es in the `.phraseapp.yml` config file.

**Comment by dragosv** [comment]
Yes, an option on the phraseapp pull is what I want as this would make transitioning from the other translation systems easier. Also, I want sometimes to only push/pull a single language, and do not want to have multiple yml files for it. And again this is something that serge.io supports and while I am implementing the PhraseApp plugin for it (https://github.com/dragosv/serge-plugins) I want this functionality.

**Comment by theSoenke** [comment]
Thanks for describing your use case. We will definitely look into this but I don't know yet when this will be available

**Comment by gaetan-petit** [comment]
Agreed that would also massively speed up push.
